registry/zookeeper/listener.go

```diff
@@ -37,18 +37,19 @@ import (
 
 // RegistryDataListener ...
 type RegistryDataListener struct {
-	interestedURL []*common.URL
-	listener      config_center.ConfigurationListener
+	subscribed map[*common.URL]config_center.ConfigurationListener
```

Comment:
is there Concurrency problem？

Reply:
i think it's unnecessary to consider Concurrency problem here.

Reply:
U should consider it.